server/persistence.ts

```diff
@@ -343,6 +344,30 @@ export class Persistence {
     this.saveData();
   }
 
+  public updateTwitchBotIntegration (twitchBotConfig: TwitchBotConfig) {
+    this.data.config = this.data.config || {};
+
+    if(!this.data.config.twitch.bot) {
+      this.data.config.twitch.bot = {
+        enabled: twitchBotConfig.enabled,
+        response: twitchBotConfig.response,
```

Comment:
will this set the default response ? if it wasn't set before?

Reply:
That endpoint is used to update the `enabled` state for the bot, and if the `bot` object didn't exist before it will set the `response` to be an empty string by default. But if the `bot` object exists it will only update the `enabled` property. That can be changes in the `updateTwitchBotIntegration` method in the `app.service.ts` file. There the default response message could be set to something like `Hey {{user}}, the commands are: {{commands}}`

Reply:
Hmm I don't have any idea where something like a default response should be added / saved - so we'll just ignore it for now